src/validateRegisterForm.test.js

```diff
@@ -26,5 +26,113 @@ describe(`Function 'validateRegisterForm':`, () => {
     expect(invalidPassword.message).toBe('Password is invalid.');
   });
 
-  // write more tests here
+  it('should return invalid message for incorrect email', () => {
+    const result = validateRegisterForm('test@com', 'P@ssword1!');
+
+    expect(result).toEqual({
+      code: 422, message: 'Email is invalid.',
+    });
+  });
+
+  it(`should return invalid message for both incorrect email
+    and password`, () => {
+    const result = validateRegisterForm('test@com', 'ssword1');
+
+    expect(result).toEqual({
+      code: 500, message: 'Password and email are invalid.',
```

Comment:
The expected status code for both invalid email and password should be `500` according to the task description, but the test case currently expects `422`. Please update the expected code to `500`.

Reply:
You're mistaken. The code expects 500 code, not 422. Review please